test/functional/lua/fs_spec.lua

```diff
@@ -340,22 +340,25 @@ describe('vim.fs', function()
   end)
 
   describe('normalize()', function()
+    local function normalize(path, opts)
+      return exec_lua([[return vim.fs.normalize(...)]], path, opts)
```

Comment:
can this be avoided by using Lua functions instead of of `vim.fn.tolower`? In https://github.com/neovim/neovim/pull/30483 I am endeavoring to revert this.

Reply:
I don't think lua is unicode-aware. Perhaps we can get away with not considering unicode for the time being?

Reply:
> Perhaps we can get away with not considering unicode for the time being?

Yeah I wouldn't worry about that. Case-insensitivity is not well-defined for non-ASCII chars. I can't find a reference, but I wonder if Windows "case insensitivity" applies to non-ASCII chars anyways.

Reply:
It does. `кириллица` is the same file as `КИРИЛЛИЦА`.